lib/cdo/shared_constants.rb

```diff
@@ -217,11 +217,9 @@ module SharedConstants
       "accelerometer.getAcceleration": null,
       "isPressed": null,
       "holdtime": null,
-      "soundSensor.start": null,
       "soundSensor.value": null,
       "soundSensor.getAveragedValue": null,
       "soundSensor.setScale": null,
-      "lightSensor.start": null,
```

Comment:
Deleting these two lines is what removes the `soundSensor.start()` and `lightSensor.start()` methods from the App Lab default toolbox.  You'll notice the keys match the `func` keys given in the droplet config.

This `APPLAB_BLOCKS` list is a constant we share across our Ruby and JavaScript code - mostly for levelbuilder purposes.

In Ruby, it gets used [in the `Applab` model](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/applab.rb#L128-L130) to [populate the default toolbox when creating a new App Lab level on levelbuilder](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/applab.rb#L77).

To use this constant in JavaScript we [run generateSharedConstants.rb before every build](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/Gruntfile.js#L701) to [generate an untracked file](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/script/generateSharedConstants.rb#L37) called sharedApplabBlocks.js.  It gets loaded in applab/levels.js and [used as the blocks for the default App Lab level](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/applab/levels.js#L35-L41), which means any level without a palette (`codeFunctions`) will get this one - that includes the [New App Lab Project.level](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/scripts/levels/New%20App%20Lab%20Project.level) which doesn't have a `codeFunctions` entry on purpose.

In summary: This list populates the toolbox for App Lab projects and for new App Lab levels on levelbuilder by referencing keys in the droplet config.